etcd/README.md

```diff
@@ -0,0 +1,60 @@
+# etcd Example
+
+## Introduction
+[etcd](https://etcd.io) is a distributed, reliable key-value store for the most
+critical data of a distributed system.
+
+In this example, we demonstrate how one can effectively and efficiently tune
+etcd resources by utilizing [the etcd benchmark tool](https://github.com/etcd-io/etcd/tree/master/tools/benchmark)
+and [prometheus](https://prometheus.io).
+
+## Prerequisites
+
+You must have a Kubernetes cluster. Additionally, you will need a local configured copy of `kubectl`.
+
+A local install of [Kustomize](https://github.com/kubernetes-sigs/kustomize/releases) (v3.1.0+) is required to manage the objects in you cluster.
```

Comment:
The last part of that should read "to manage the objects in ~~you~~ your cluster."